### PR TITLE
(tlg0070)-(tlg0591)_(tlg0593)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts_.xml data and other metadata to current standards for a number of files.

### CODE REVIEW

File: data/tlg0591/1st1K002/tlg0591.1st1K002.1st1K-grc1.xml [hunk 1]
```diff
@@ -4,10 +4,9 @@
   <teiHeader xml:lang="eng">
     <fileDesc>
   <titleStmt>
-    <title>Odysseus</title>
     <title xml:lang="grc">ΟΔΥΣΣΕΥΣ</title>
```
Comment (lcerrato) [inline data/tlg0591/1st1K002/tlg0591.1st1K002.1st1K-grc1.xml:7]: Ὀδυσσεύς

**Review by lcerrato — CHANGES_REQUESTED** [review]
I would not add titles that are all caps in Greek since that means we lose accentuation. Getting the accented titles can require some digging, though, so if they aren't obvious, you could just flag me and I can see what I can come up with. 
I do generally avoid all caps though as these aren't parsable.
So let me know what you want to do with this batch.

File: data/tlg0591/1st1K002/tlg0591.1st1K002.1st1K-grc1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0591/1st1K002/tlg0591.1st1K002.1st1K-grc1.xml:7]: You know I think the Greek title was already in the file like that and I just deleted the English, but I will fix that! In general I'm trying not to add too many titles in any language that don't show up in the source itself!

File: data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml [hunk 2]
```diff
@@ -4,10 +4,9 @@
   <teiHeader xml:lang="eng">    
     <fileDesc>
       <titleStmt>
-        <title>Encomium of Helen</title>
         <title xml:lang="grc">ΕΛΕΝΗΣ ΕΓΚΩΜΙΟΝ</title>
```
Comment (lcerrato) [inline data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml:7]: Encomium of Helen (Ἑλένης ἐγκώμιον), Defense of Palamedes (Ὑπέρ Παλαμήδους ἀπολογία), On Non-Existence (Περὶ τοῦ μὴ ὄντος ἢ Περὶ φύσεως), and Epitaphios (Επιτάφιος)

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml]: Thanks! I've pulled your changes into my branch! Is there anything else I need to do @lcerrato

Comment (lcerrato) [reply data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml]: @AlisonBabeu 
Just check my work. Did we catch all of them?

**Comment by lcerrato** [comment]
@AlisonBabeu I spotted a minor typo in the last title but once this set of checks is done I think it's good.

**Review by AlisonBabeu — COMMENTED** [review]
It really appears that you have made all the changes! @lcerrato

**Pull request merged**